shared/fs/header/header.native.js

```diff
@@ -1,68 +1,27 @@
 // @flow
 import * as React from 'react'
-import {styleSheetCreate, collapseStyles, globalColors, globalStyles, globalMargins} from '../../styles'
-import {BackButton, Box, Icon, Text} from '../../common-adapters'
+import * as Kb from '../../common-adapters'
 import AddNew from './add-new-container'
 import {type FolderHeaderProps} from './header'
 
 const Header = ({title, path, onBack, onChat}: FolderHeaderProps) => (
-  <Box style={styles.stylesFolderHeaderContainer}>
-    <Box style={styles.stylesFolderHeaderRow}>
-      <BackButton onClick={onBack} style={!onBack ? styles.onBackHidden : undefined} />
-      <Box style={styles.stylesFolderHeaderRoot}>
-        <Text center={true} type="BodySemibold" style={styles.stylesTitle}>
-          {title}
-        </Text>
-      </Box>
-      <Box style={styles.stylesAddNewBox}>
-        <AddNew path={path} style={styles.stylesIcons} />
-      </Box>
-      {onChat && (
-        <Box style={styles.stylesAddNewBox}>
-          <Icon
-            type="iconfont-chat"
-            style={collapseStyles([styles.stylesIcons])}
-            color={globalColors.black_50}
-            fontSize={22}
-            onClick={onChat}
-          />
-        </Box>
-      )}
-    </Box>
-  </Box>
+  <Kb.HeaderHocHeader
```

Comment:
Is this called HeaderHocHeader because it's usually used in HeaderHoc, or does it have other implications?